src/schemas/v2/combined.json

```diff
@@ -36632,6 +36632,39 @@
               }
             ]
           }
+        },
+        "documents": {
+          "title": "Documents",
+          "description": "Documents associated with the transaction",
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "displayName": {
+                "title": "Human-readable name of document",
+                "type": "string",
+                "minLength": 1
+              },
+              "fileName": {
+                "title": "File name of the document",
+                "type": "string",
+                "minLength": 1
+              },
+              "mimeType": {
+                "title": "Document MIME type",
+                "type": "string",
+                "minLength": 1
+              },
+              "documentType": {
+                "title": "Human-readable document category",
+                "type": "string",
+                "minLength": 1
+              },
+              "externalIds": {
```

Comment:
Where is the actual document stored, 
should we either provide a field for inline base64 document (like in verified claims)
or provide URL for the document, 

OPTIONAL Addition Need to discuss-> with URL can add fields to provide basic auth type (that signifies, if URL is publicly available or need some Auth).

Reply:
And still keep an additional "externalReference" `string` or `object` to capture custom metadata

Reply:
Our view is that the verification + evidence part of the claim provides the location of the file